src/lib/notifications/backgroundActions.native.js

```diff
@@ -10,28 +9,33 @@ export const CLAIM_NOTIFICATION = 'claim-notification'
 
 export const dailyClaimTime = new Date().setUTCHours(12)
 
+export const notificationsCategories = {
+  CLAIM: 'CLAIM',
+}
+
 export const dailyClaimNotification = async (userStorage, goodWallet) => {
   try {
-    const { entitlement: dailyUBI } = await goodWallet.getClaimScreenStatsFuse()
+    const dailyUBI = await goodWallet.checkEntitlement()
 
     // We should notify once: only in first bg-fetch call after daily claim time
-    const lastClaimNotification = await userStorage.userProperties.get(LAST_CLAIM_NOTIFICATION)
+    const lastClaimNotification = await userStorage.userProperties.get('lastClaimNotification')
     const needToNotify = dailyUBI && Date.now() >= dailyClaimTime && lastClaimNotification < dailyClaimTime
 
     if (needToNotify) {
       Notifications.postLocalNotification({
         title: t`Your daily UBI Claim is ready!`,
-        body: t`You can claim your daily GoodDollar UBI`,
+        body: dailyUBI,
```

Comment:
that doesnt make sense, please consult with @patpedrosa on all texts.
1. dailyUBI is in cents
2. doesnt have G$ sign